### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dwk2**
Updating maintainer's list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,21 @@
 # Islandora ISLE Interest Group
-The ISLE Interest Group consists of ISLE Maintainers, beta testers, and interested members of the community. 
+The ISLE Interest Group consists of [ISLE Maintainers](https://github.com/Islandora-Collaboration-Group/ISLE#isle-project-maintainers), beta testers, and interested members of the community. 
 
 ## Conveners
-* Benjamin Rosner (Barnard) 
-* Shaun Trujillo (MHC)
+* Shaun Trujillo, Mount Holyoke College
```
Comment (bseeger) [inline README.md]: Is this still true?  :-)   Just checking in.

**Comment by bseeger** [comment]
Who has merge power in this repo?    Thanks!

**Comment by manez** [comment]
@bseeger merge power belongs to: @dwk2 @shauntru @br2490 @marksandford and @exsilica 

I'd be happy to update that list if there are changes to the group (I assume you should be on there too?)

File: README.md (hunk 1 above)
Comment (dwk2) [reply README.md]: Yes, still true.

**Comment by manez** [comment]
@dwk2 this pr has conflicts now so I can't merge. Sorry!

**Comment by dwk2** [comment]
This PR is now obsolete. No longer needed. Closing PR. -- David KC

**Pull request closed without merging**